test/e2e/singlecluster/pod_test.go

```diff
@@ -82,24 +81,22 @@ var _ = ginkgo.Describe("Pod groups", func() {
 		})
 
 		ginkgo.It("should admit group that fits", func() {
-			g1 := podtesting.MakePod("g1", ns.Name).
+			group := podtesting.MakePod("group", ns.Name).
 				Image("gcr.io/k8s-staging-perf-tests/sleep:v0.0.3", []string{"1s"}).
 				Queue(lq.Name).
 				Request(corev1.ResourceCPU, "1").
 				MakeGroup(2)
-			g1Key := client.ObjectKey{Namespace: ns.Name, Name: "g1"}
-			for _, p := range g1 {
+			gKey := client.ObjectKey{Namespace: ns.Name, Name: "group"}
+			for _, p := range group {
 				gomega.Expect(k8sClient.Create(ctx, p)).To(gomega.Succeed())
 				gomega.Expect(p.Spec.SchedulingGates).
 					To(gomega.ContainElement(corev1.PodSchedulingGate{
 						Name: pod.SchedulingGateName}))
 			}
 			ginkgo.By("Starting admission", func() {
-				util.UnholdQueue(ctx, k8sClient, cq)
```

Comment:
In my initial approach, I was blocking the queue on creation. I abandoned that approach, so this line was unnecessary.